### ISSUE TITLE
Power Sensor not closing when shutting down TV

### ISSUE THREAD

**Issue by MirkoKas** [grzegorz914/homebridge-lgwebos-tv#173]
I just bought a new C3 TV and as the HomeKit integration by LG itself isn't perfect I tried your plugin (thanks for building and maintaining it!).
Everything seems to work, but only the power sensor I have is not closing when I put the TV into stand by. It's stays open until I restart the plugin in homebridge. No difference how I turn it off - HomeKit, TV remote, CEC. Always the same.

Here's the log snippet of homebridge - hope it helps:

[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Set Command Name: Power, Reference: POWER
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active Standby","processing":"Prepare Power On","onOff":"on","reason":"remoteKey"}}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active Standby",
  "processing": "Prepare Power On",
  "onOff": "on",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active Standby",
  "processing": "Prepare Power On",
  "onOff": "on",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active Standby","processing":"LastInput Ready","onOff":"on","reason":"remoteKey"}}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active Standby",
  "processing": "LastInput Ready",
  "onOff": "on",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active Standby",
  "processing": "LastInput Ready",
  "onOff": "on",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active Standby","processing":"Screen On","onOff":"on","reason":"remoteKey"}}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active Standby",
  "processing": "Screen On",
  "onOff": "on",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active Standby",
  "processing": "Screen On",
  "onOff": "on",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"3b53ed9e0000","payload":{"subscribed":true,"returnValue":true,"appId":"com.webos.app.hdmi1","processId":"","windowId":""}}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 3b53ed9e0000, Data: {
  "subscribed": true,
  "returnValue": true,
  "appId": "com.webos.app.hdmi1",
  "processId": "",
  "windowId": ""
}
[9/5/2023, 12:08:17 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: App: {
  "subscribed": true,
  "returnValue": true,
  "appId": "com.webos.app.hdmi1",
  "processId": "",
  "windowId": ""
}
[9/5/2023, 12:08:19 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:19 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:19 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:19 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:20 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active"}}
[9/5/2023, 12:08:20 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active"
}
[9/5/2023, 12:08:20 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active"
}
[9/5/2023, 12:08:22 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:22 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:22 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:22 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:24 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:24 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:24 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:24 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:27 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:27 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:27 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:27 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:29 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:29 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:29 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:29 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:32 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:32 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:32 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:32 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:34 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:34 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:34 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:34 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:37 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:37 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:37 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:37 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:39 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:39 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:39 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:39 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:42 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:42 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:42 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:42 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Set Command Name: Power, Reference: POWER
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active","processing":"Request Power Off","onOff":"off","reason":"remoteKey"}}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active",
  "processing": "Request Power Off",
  "onOff": "off",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active",
  "processing": "Request Power Off",
  "onOff": "off",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active","processing":"Request Active Standby","onOff":"off","reason":"remoteKey"}}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active",
  "processing": "Request Active Standby",
  "onOff": "off",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active",
  "processing": "Request Active Standby",
  "onOff": "off",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active","processing":"Prepare Active Standby","onOff":"off","reason":"remoteKey"}}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active",
  "processing": "Prepare Active Standby",
  "onOff": "off",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active",
  "processing": "Prepare Active Standby",
  "onOff": "off",
  "reason": "remoteKey"
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"3b53ed9e0000","payload":{"subscribed":true,"returnValue":true,"appId":"","processId":"","windowId":""}}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 3b53ed9e0000, Data: {
  "subscribed": true,
  "returnValue": true,
  "appId": "",
  "processId": "",
  "windowId": ""
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: App: {
  "subscribed": true,
  "returnValue": true,
  "appId": "",
  "processId": "",
  "windowId": ""
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Received message: {"type":"response","id":"7ae193bd0000","payload":{"returnValue":true,"state":"Active Standby"}}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Message Id: 7ae193bd0000, Data: {
  "returnValue": true,
  "state": "Active Standby"
}
[9/5/2023, 12:08:43 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Power: {
  "returnValue": true,
  "state": "Active Standby"
}
[9/5/2023, 12:08:44 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:44 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:44 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:44 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:47 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:47 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:47 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:47 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:49 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:49 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.
[9/5/2023, 12:08:49 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket send heartbeat.
[9/5/2023, 12:08:49 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Specialized socket received heartbeat.
[9/5/2023, 12:08:52 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked send heartbeat.
[9/5/2023, 12:08:52 PM] [homebridge-lgwebos-tv] Device: 192.168.99.20 LG TV C3, debug: Socked received heartbeat.

**Comment by MirkoKas** [comment]
Quick update:
I just saw within the logs that the connection got disconnected a couple of minutes after shutting the tv off. The socket connection got re-connected only couple of seconds later. Again, about 15min later the connection got disconnected again and stays this way. The Power sensor changes accordingly. I noticed this because my automation I trigger when the TV goes off started. A couple of seconds, the automation for turning the TV on triggered. At last, the automation for shutting off triggered again 15min later.

I'm new to OLED TVs. Could it be related to that Pixel refresh stuff that is going on when the TV is in stand by?
